float64[] path_X
float64[] path_Y
float64[] path_Z
float64[] start_state
float64 hdg
float64 psi
